### PR TITLE
Bug 1188069 - Refactor to move RTL overrides next to LRT/defaults

### PR DESCRIPTION
**Pull request by sfoster**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=7bfb362e026fe3b831d84b8b57ccfb6d189ee9d2

File: apps/ftu/style/style.css [hunk 1]
```diff
@@ -74,6 +74,18 @@ a {
...
 /*
+ * Navigation Buttons
+ *
+ * right=forward, left=back in both LTR and RTL
+ */
+
+html[dir="ltr"] gaia-buttons.nav,
+html[dir="rtl"] gaia-buttons.nav {
+  direction: ltr;
+  unicode-bidi:embed;
```
Comment (anefzaoui) [inline apps/ftu/style/style.css]: nit: space after colon

**Comment by anefzaoui** [comment]
/me is just realizing he can only comment on changes in a PR.
So I'll just post my comments on unchaged code here..

Line 489:
I don't know if the file you're reading the new spec from has this or not, but part of the pattern we agreed on internally before is reducing what's redundant. Where for example
`padding: 1.5rem 1.5rem 0.5rem 1.5rem;`
Should become
`padding: 1.5rem 1.5rem 0.5rem;`
Totally up to you, just had to chime in.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=0a871813fc3c1136adaa001ffad4ce55da1d4dd1

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=0a871813fc3c1136adaa001ffad4ce55da1d4dd1

**Pull request merged**